src/OpenTelemetry/Metrics/Exemplar/IExemplarFilter.cs

```diff
@@ -0,0 +1,62 @@
+// <copyright file="IExemplarFilter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+namespace OpenTelemetry.Metrics;
+
+/// <summary>
+/// The interface defining Exemplar Filter.
+/// </summary>
+internal interface IExemplarFilter
```

Comment:
Is "filter" coming from the spec? Asking because `IExemplarSampler`, `AlwaysOnExemplarSampler`, & `AlwaysOffExemplarSampler` might fit in better with what we have for tracing 🤷

Reply:
https://github.com/open-telemetry/opentelemetry-specification/pull/2919 I am proposing fixes in spec in parallel.